ldms/src/ldmsd/ldmsd_updtr.c

```diff
@@ -1482,6 +1482,7 @@ int ldmsd_updtr_prdcr_add(const char *updtr_name, const char *prdcr_regex,
 	if (rc) {
 		rc = EINVAL;
 		regfree(&prd_match->regex);
```

Comment:
We don't need to call regfree() if regcomp() fails.

Reply:
While the man page doesn't explicitly your regfree claim, the glibc regcomp source seems to be consistent with it. I'll delete the regfree call in this error path.

Reply:
> While the man page doesn't explicitly your regfree claim, the glibc regcomp source seems to be consistent with it. I'll delete the regfree call in this error path.

It doesn't break anything, it just doesn't do anything. I'm fine with it either way. I was just letting you know.